### DISCUSSION TITLE
Show exact class name/inheritance hierarchy for Call method track in AnimationPlayer

### DISCUSSION THREAD

**Discussion by Kartopod** [godotengine/godot-proposals#10402] [Editor]
When you create a key on a CallMethodTrack and select a node that has a script that inherits another user script, all user-made methods show up under a single section called 'Script Methods'. 

The engine methods are already sorted based on class names, so it doesn't make sense that user made methods don't also do the same. This get's especially unusable when using virtual methods, since the same names appear multiple times, and the user has no way to know which class the method belongs to. 

![image](https://github.com/user-attachments/assets/d3a47f9f-ece3-4add-8f74-97b955146259)

**Comment by dalexeev** [comment]
* I noticed this while working on godotengine/godot#80411.
* This is fixed in godotengine/godot#80418, but it needs to be finished.
